rplugin/python3/molten/moltenbuffer.py

```diff
@@ -215,11 +226,16 @@ def tick(self) -> None:
             starting_status = output.status
             did_stuff = self.runtime.tick(output)
 
-            if starting_status != OutputStatus.DONE and output.status == OutputStatus.DONE:
+            if (
+                starting_status != OutputStatus.DONE
+                and output.status == OutputStatus.DONE
+            ):
                 if self.options.auto_open_html_in_browser:
                     self.open_in_browser(silent=True)
                 if self.options.auto_image_popup:
                     self.open_image_popup(silent=True)
+                if isinstance(self.canvas, WeztermCanvas):
```

Comment:
Similar, I don't think this should have to be here, opening images should just happen in the canvas.